res/ktopt_moveit_parameters.yaml

```diff
@@ -1,19 +1,41 @@
 ktopt_interface:
   num_iterations: {
     type: int,
-    description: "dummy",
+    description: "Number of iterations for the Drake mathematical program solver.",
     default_value: 1000,
     validation: {
       gt_eq<>: [1]
     }
```

Comment:
Actually, I realize this parameter is not currently being used, as we're using default options in the mathematical program `Solve()` function.